media_kit_native_event_loop/pubspec.yaml

```diff
@@ -28,3 +28,18 @@ flutter:
         ffiPlugin: true
       ios:
         ffiPlugin: true
+
+      # NOTE: package:media_kit ships pre-built media_kit_native_event_loop shared libraries on Android within package:media_kit_libs_android_audio & package:media_kit_libs_android_video packages along side libmpv/FFmpeg shared libraries.
+      # We do not build on dependent's machine because it requires setting up NDK etc. & build process is rather complex.
+      #
+      # For building a `libmedia_kit_native_event_loop.so` using this CMake:
+      # 1. Generate the plugin_ffi template project for Android using:
+      #    * flutter create --org com.alexmercerind --template plugin_ffi --platforms=android -a java .
+      # 2. Add the libmpv/FFmpeg & other dependencies to which this target will link at:
+      #    * ../android/libs/lib/arm64-v8a/*
+      #    * ../android/libs/lib/armeabi-v7a/*
+      #    * ../android/libs/lib/x86_64/*
+      # 3. Add the libmpv headers at include/*
+      # 4. Generate the APK file:
+      #    * flutter build apk --release
+      # 5. Extract the `libmedia_kit_native_event_loop.so` for each ABI & architecture.
```

Comment:
Maybe the build instructions can be moved to the repo containing the jars, and just add a link to that repo here?